service/src/main/resources/db/changelog.xml

```diff
@@ -25,4 +25,6 @@
     <include file="changesets/20230922_initial_schema.yaml" relativeToChangelogFile="true"/>
     <include file="changesets/20231116_output_attribute.yaml" relativeToChangelogFile="true"/>
     <include file="changesets/20240304_selector_column.yaml" relativeToChangelogFile="true"/>
+    <include file="changesets/20240326_remove_criteria_group_columns.yaml" relativeToChangelogFile="true"/>
```

Comment:
What is this? Is it missing?

Reply:
Yeah, it was missing from a [previous PR](https://github.com/DataBiosphere/tanagra/pull/788).